crates/sui-verifier-transactional-tests/tests/private_generics/no_public_transfer.move

```diff
@@ -10,21 +10,21 @@ module a::m {
     struct S has key { id: sui::object::UID }
 }
 
-//# publish
+//# publish --dependencies a
```

Comment:
These changes were motivated by the fact that because we now fetch packages before running verification, we can get a missing dependency error before we get the error from the sui verifier about transfer.

Should be a fairly simple fix to say that if the upgrades flag is not enabled, we redundantly run verification first in the publish flow -- hopefully, if we re-genesis again, we can get rid of that weird set-up.